sim/global-types.ts

```diff
@@ -250,7 +250,7 @@ interface EventMethods {
 	onNegateImmunity?: ((this: Battle, pokemon: Pokemon, type: string) => boolean | void) | boolean
 	onOverrideAction?: (this: Battle, pokemon: Pokemon, target: Pokemon, move: ActiveMove) => string | void
 	onPrepareHit?: CommonHandlers['ResultSourceMove']
-	onRedirectTarget?: (this: Battle, target: Pokemon, source: Pokemon, source2: Effect, move: ActiveMove) => Pokemon | void
+	onRedirectTarget?: (this: Battle, target: Pokemon, source: Pokemon, source2: Effect, move: ActiveMove) => Pokemon | Pokemon[] | void
```

Comment:
The relayVar is of type `Pokemon`, so this return type will actively cause crashes if someone ever adds a lower-priority `RedirectTarget` handler.